pkg/crc/machine/start.go

```diff
@@ -568,7 +568,7 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 			}
 		}
 		logging.Info("Adding microshift context to kubeconfig...")
-		if err := mergeKubeConfigFile(constants.KubeconfigFilePath); err != nil {
+		if err := mergeKubeConfigFile(constants.KubeconfigFilePath, ""); err != nil {
```

Comment:
For what it's worth, adding a dummy argument to a function in the "main" codepath because it will be used in unit testing is not great for readability. One may wonder "What does "" stand for?" while reading the code.

You can achieve the same with an intermediate function 
```
func mergeToGlobalKubeConfigFile(kubeConfigFile string) error {
	return mergeKubeConfigFile(kubeConfigFile, getGlobalKubeConfigPath())
}
```
And remove the special handling of `""` in `mergeKubeConfigFile`.

Reply:
Thanks, I'll make this change.